scripts/extensions/sams/src/components/assets/uploadAssetModal.tsx

```diff
@@ -186,7 +190,7 @@ export class UploadAssetModalComponent extends React.Component<IUploadAssetModal
 
         return (
             <FileUploadModal
-                dark={true}
+                dark={false}
```

Comment:
Why was this changed from dark to light?

Reply:
The tags were not being visible in dark.